src/style.css

```diff
@@ -1,3 +1,85 @@
+/* stylelint-disable block-no-empty */
+html {
+  box-sizing: border-box;
+}
+
 body {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  font-family: Roboto, Arial, sans-serif;
+  font-weight: 500;
+  font-style: normal;
+  font-size: 12px;
+}
+
+ul,
+li {
   margin: 0;
+  padding: 0;
+}
+
+.header {
+  height: 60px;
+}
```

Comment:
it's not a good practice to hardcode height for elements with content inside. Let's use padding for your links instead
```suggestion
```